code/_global_vars/lists/adminhelp.dm

```diff
@@ -0,0 +1,2 @@
+//This is a list of words which are ignored by the parser when comparing message contents for names. MUST BE IN LOWER CASE!
+var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","alien","as")
```

Comment:
This isn't a managed global, you create them with defines, like

GLOBAL_LIST_INIT(adminhelp_ignored_words, list("unknown","the","a","an","of","monkey","alien","as"))

Reply:
Will go up in the next commit.